packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -266,10 +266,28 @@ addGraphQLResolvers({
           rating: 0
         }
       })
-    }
+    },
+    async UsersReadPostsOfTargetUser(root: void, { userId, targetUserId, limit = 20 }: { userId: string, targetUserId: string, limit: number }, context: ResolverContext) {
+      const { currentUser, repos } = context
+      if (!currentUser) {
+        throw new Error('Must be logged in to view read posts of target user')
+      }
+
+      const posts = await repos.posts.getUsersReadPostsOfTargetUser(userId, targetUserId, limit)
+
+      return posts
+    }, 
   },
 })
 
+
+addGraphQLQuery("UsersReadPostsOfTargetUser(userId: String!, targetUserId: String!, limit: Int): [Post]");
```

Comment:
Probably the return type should be `[Post!]` (indicating that the returned array shouldn't ever have `null` values in it, which is allowed by `[Post]` but not `[Post!]`)